README.md

```diff
@@ -61,132 +59,115 @@ maintaining the usual way of interacting with it.
 
-### Easy Scaling
-The production of ML products often involves several stages each of which requires different computing power.
 
-This solution allows Slurm to reuse the unique Kubernetes' ability to scale automatically depending on the current
-needs. You can simply change a single value in the YAML manifest, and watch the cluster changes in size.
+### GPU health checks
+If your Kubernetes cluster contains NVIDIA GPUs, the operator will perform regular GPU health checks. If a Slurm
+node fails a health check, the operator “drains” it, so that new jobs are not scheduled on it.
 
 
-### High Availability
-Kubernetes brings some level of HA out of the box. If a Pod or container dies (e.g. Slurm controller), Kubernetes
-recreates it.
+### Easy scaling
+Each stage of building an ML product has its own requirements for computing resources.
 
-Our operator improves this further, continuously bringing the entire cluster to the desired state.
+Soperator allows Slurm to reuse the unique Kubernetes feature of scaling automatically according to current needs.
+You can simply change a single value in the YAML manifest and watch the cluster change in size.
 
 
-### Isolation of User Actions
-Users can’t unintentionally break the Slurm cluster itself - all their actions are isolated within a dedicated
-environment (some sort of container). This clearly defines the boundary between the operator's responsibility and the
-users' one.
+### High availability
+Kubernetes comes with some level of HA out of the box. If a pod or container, such as a Slurm controller, fails,
+Kubernetes recreates it.
 
+Soperator takes this even further, continuously bringing the entire cluster up to the desired state.
 
 
-### Observability
-This solution implements integration with a monitoring stack that can be installed separately (though we provide Helm
-charts for that). It consists of gathering various Slurm statistics and hardware utilization metrics. Users can observe
-this information on the dashboards we provide as well.
-
+### Isolation of user actions
+All user actions are isolated within a dedicated container-like environment, so that an action can't break the
+Slurm cluster itself by accident. This defines a clear boundary between operator and user responsibility.
 
 
 ## ❌ Limitations
-- **GPUs are required**. Although supporting CPU-only clusters or partitions seems pretty straightforward, we haven’t
-  done it yet.
-- **Scaling clusters down**. Only scaling up works flawlessly. Scaling down remains deleted nodes in the controller
-  view. However, they can be removed manually using `scontrol`.
-- **Single-partition cluster**. The Slurm's ability to split clusters into several partitions isn't supported now.
+- **GPUs are required**. Although support for CPU-only clusters or partitions seems pretty straightforward, we haven't
+  implemented it yet.
+- **Unreliable scale-downs**. Clusters only scale up correctly. Scaling down leaves deleted nodes in the controller
```

Comment:
It's not actually "unreliable". It doesn't break anything. Maybe we can drop this limitation at all.
It's just something that may bother someone's eyes - no more